corehq/apps/hqwebapp/urls.py

```diff
@@ -46,6 +46,18 @@
     TwoFactorSetupView,
 )
 
+
+def two_factor_urls():
+    def get_urls(urls):
+        if isinstance(urls, list):
+            # django-two-factor-auth==1.6.2
+            return urls
+        # django-two-factor-auth~=1.12
+        assert isinstance(urls, tuple) and isinstance(urls[0], list), urls
+        return urls[0]
+    return get_urls(tf_urls) + get_urls(tf_twilio_urls), 'two_factor'
```

Comment:
nice shim. There's probably a cleanup step once we're on the other side of this, but i like the approach